### PR TITLE
Fix license to the SPDX License format

### PR DESCRIPTION
**Pull request by execat**
Removes the following warning by adding license data from http://spdx.org/licenses/Unlicense.html:

screen shot 2015-10-30 at 2 39 40 pm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "email": "todd@twolfson.com",
     "url": "http://twolfson.com/"
   },
+  "license": "Unlicense",
```
Comment (twolfson) [inline package.json]: Can you relocate this to the existing `licenses` position and remove `licenses`?

**Comment by twolfson** [comment]
Awesome, thanks =) Left a request to replace the deprecated `licenses` key in a more in-place fashion

**Comment by execat** [comment]
@twolfson Please check again.

**Comment by twolfson** [comment]
Looks good. I should land this tonight or tomorrow. Thanks for the patch =)

**Comment by twolfson** [comment]
This has been merged/released in `2.2.1`. Thanks again for the patch =)

**Comment by execat** [comment]
Thanks for the quick review and merge!
Long live the Google Music Electron!

**Pull request merged**